### DISCUSSION TITLE
Change isMultiCamSessionEnabled on live ?

### DISCUSSION THREAD

**Discussion by emirbeytekin** [HaishinKit/HaishinKit.swift#1435] [Q&A]
Hello,
I want to instantly update the multiCam enabled property to true / false by adding a button in my Stream application. Is this possible ?

**Comment by shogo4405** [comment]
It's not possible. If you wish to do something like that, it's recommended to have it set to true from the beginning. I suggest attaching a second camera halfway through instead.

**Comment by emirbeytekin** [comment]
I created a solution for this
I start broadcasting, then I can switch from multi camera mode to single camera mode, then I can switch back to multi camera mode if I want
